docs/source/conf.py

```diff
@@ -14,16 +14,26 @@
 # import sys
 # sys.path.insert(0, os.path.abspath('.'))
 
+import os
+from datetime import datetime
+
 import sphinx_rtd_theme
 
 # -- Project information -----------------------------------------------------
 
 project = 'testbed'
```

Comment:
Shouldn't be the project name `cohydra`?

Reply:
Yes, it will be cohydra. But in order to have clearer structure, there will be another PR renaming all the `testbed` or `sn3t`s to `cohydra`.

Reply:
Okay, thanks!